companion/src/modeledit/telemetry.cpp

```diff
@@ -324,6 +325,13 @@ TelemetryCustomScreen::TelemetryCustomScreen(QWidget *parent, ModelData & model,
     connect(minSB[l], SIGNAL(valueChanged(double)), this, SLOT(barMinChanged(double)));
     ui->screenBarsLayout->addWidget(minSB[l], l, 1, 1, 1);
 
+    minTime[l] = new TimerEdit(this);
+    minTime[l]->setProperty("index", l);
+    minTime[l]->setProperty("type", "min");
+    connect(minTime[l], SIGNAL(editingFinished()), this, SLOT(barTimeChanged()));
+    ui->screenBarsLayout->addWidget(minTime[l], l, 1, 1, 1);
+    minTime[l]->hide();
+
     QLabel * label = new QLabel(this);
     label->setAutoFillBackground(false);
     label->setStyleSheet(QString::fromUtf8("Background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 128, 255), stop:0.339795 rgba(0, 0, 128, 255), stop:0.339799 rgba(255, 255, 255, 255), stop:0.662444 rgba(255, 255, 255, 255),)\n"""));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Merge this literal with previous. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS3728)